### ISSUE TITLE
Kapitel 7 Bauverfahren

### ISSUE THREAD

**Issue by mme1950** [ASJeton/Klassenvorschrift#3]
Hier haben wir ein kleines Problem mit dem "Sichtcarbon" auf dem Schwerttisch.

Klärung erforderlich: Ist das durch Verstärkung jeglicher Art abgedeckt?

**Comment by daniel-richter** [comment]
Würde aber 7.1.3 nicht auch die Materialien für "Versteifungen jeglicher Art" einschränken? D.h. ich könnte zwar beliebige Versteifungen einbauen, dürfte dafür aber trotzdem immer noch nur Glasfaser, Polyester, Hartschaum und Holz verwenden?

>    7.1.1 Sandwichbauweise ist erlaubt. Sandwichbauweise darf auch zur Verstärkung von einzelnen Flächen eingesetzt werden.
>    7.1.2 Schotten, Reitbalken sowie Versteifungen jeglicher Art sind freigestellt.
>    7.1.3 Als Materialien sind ausschließlich zugelassen: Glasfaser, Polyester, Hartschaum, Holz.

**Comment by mme1950** [comment]
Ja, 7.1.3 soll sich wahrscheinlich auch auf die Versteifungen beziehen. Das hatte ich übersehen.
Ich wollte ja mit dem technischen Ausschuss vom DSV abstimmen, wie man das am besten macht.

**Comment by mme1950** [comment]
@daniel-richter
Ich habe heute mit dem Vorsitzenden der Abteilung Technik im DSV gesprochen.
Er empfiehlt, den Schwertkastendeckel/Schwerttisch explizit zu regeln, und den Einsatz von Kohle auf Sicht-Carbon zu beschränken. Ein Schwerttisch ist auch keine Versteifung im eigentlichen Sinne.

Aber, was 7.1.2 und 7.1.3 betrifft, ist die Formulierung auf den ersten Blick widersprüchlich. Aber 7.1.2 bezieht sich auf die Art der Versteifungen, 7.1.3 auf die Materialien. Die Versteifungen jeglicher Art betreffen wesentlich die Bauweise ab Werft. Unklar ist, ob die eingelegten Bleche nicht damit im Widerspruch sind. OK - sind wir mal nicht päpstlicher als der Papst.
Vielleicht lassen wir die beiden Absätze wie sie sind.
